test/e2e/cross-account/smoke.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="$THIS_DIR/../../.."
+SCRIPTS_DIR="$ROOT_DIR/scripts"
+
+source "$SCRIPTS_DIR/lib/common.sh"
+source "$SCRIPTS_DIR/lib/k8s.sh"
+source "$SCRIPTS_DIR/lib/testutil.sh"
+
+
+# AWS_ACCOUNT_ID_2 should be configured to allow cross account resources management
+# from AWS_ACCOUNT_ID
+AWS_ACCOUNT_ID_2=""
```

Comment:
I'm still working on this script. I need to figure out a way to switch accounts for github action steps

Reply:
Note that the `aws` CLI tool supports named profiles for exactly this purpose:

https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html

Reply:
Oh yeah let's use this!